examples/stm32f334/src/bin/pwm.rs

```diff
@@ -27,7 +27,10 @@ async fn main(_spawner: Spawner) {
         config.rcc.ahb_pre = AHBPrescaler::DIV1;
         config.rcc.apb1_pre = APBPrescaler::DIV2;
         config.rcc.apb2_pre = APBPrescaler::DIV1;
+
+        // TODO: The two lines here do the same thing
         config.rcc.hrtim = HrtimClockSource::PllClk;
+        config.rcc.mux.hrtim1sw = Some(embassy_stm32::pac::rcc::vals::Timsw::PLL1_P);
```

Comment:
Not sure what to do here. Both of these modify the same system, but I don't want to remove the old API. 
In addition, the `Timsw` enum does seem to be easily importable, something should probably be done for that.

Reply:
yes, please remove the old API. breaking changes are fine.

about imports: yes, they shouldn't require `embassy_stm32::pac`... perhaps you can generate a `mod mux` and put it in `embassy_stm32::rcc::mux`, so you'd do `embassy_stm32::rcc::mux::Timsw::PLL1_P`. I wouldn't put all the enums in the main `embassy_stm32::rcc` mod because bigger chips have lots of them, which will clutter docs.

Reply:
Should I re-export the enums from `pac` or make new ones that reference the values from `pac`? If I should do the latter, should I also use the same names, or should I make better names?

Reply:
reexporting is fine, RCC already reexports many enums from the PAC like for PLL config etc. 

I don't tihnk it'd be easy to make build.rs automatically come up with better names? i'd just use the same names.